src/ontology/mondo-ingest.Makefile

```diff
@@ -177,6 +171,22 @@ metadata/mondo.sssom.config.yml:
 
 mappings: sssom $(ALL_MAPPINGS)
 
+.PHONY: unmapped-terms-docs
+unmapped-terms-docs: $(foreach n,$(ALL_COMPONENT_IDS), reports/$(n)_unmapped_terms.tsv)
+	python3 $(SCRIPTSDIR)/unmapped_docs.py
+
+.PHONY: unmapped-terms-tables
+unmapped-terms-tables: $(foreach n,$(ALL_COMPONENT_IDS), reports/$(n)_unmapped_terms.tsv)
+
+$(REPORTDIR)/%_unmapped_terms.tsv: $(REPORTDIR)/%_term_exclusions.txt $(REPORTDIR)/mirror_signature-%.tsv $(TMPDIR)/mondo.sssom.tsv metadata/%.yml $(COMPONENTSDIR)/%.owl
+	python3 $(SCRIPTSDIR)/unmapped_tables.py \
+	--exclusions-path $(REPORTDIR)/$*_term_exclusions.txt \
+	--mirror-signature-path $(REPORTDIR)/mirror_signature-$*.tsv \
+	--sssom-map-path $(TMPDIR)/mondo.sssom.tsv \
+	--onto-config-path metadata/$*.yml \
+	--onto-path components/$*.owl \
```

Comment:
@matentzn Double checking. You said [in this issue](https://github.com/monarch-initiative/mondo-ingest/issues/102) that I should query a `tmp/mondo-ingest.db` for labels. Is that correct? If so, I need to change what I'm doing.

Instead, right now I'm querying the `components/*.owl` file using SPARQL. Originally I was trying to get a `.db` file for each ontology. I still want to do that, but until I [clear up an/some issue(s)](https://github.com/monarch-initiative/mondo-ingest/issues/136#issuecomment-1402980219).

Reply:
These details are too complicated for me to keep in my mind. Think of it like this:

1. We want track progress relative to the _original unaltered_ sources
2. components are altered (i.e. they are missing many terms)
3. mondo-ingest contains components, therefore altered

Try to figure out whats right from this! Understand the problem, rather than trying to stitch together pipelines based on faulty stuff I say all the time :)

Reply:
No problem Nico. I know what to do here. 

(1) and (2) are not a problem based on the fact that I'm using `mirror_signature` to capture the list of concepts. My only consideration for using `components/` is to help me create a `semsql` db without it erroring. But honestly if it has less labels to pull from I might as well keep using SPARQL directly on the `mirror/` files.

For (3), yeah I agree I was just confused. Sometimes I take what you say as sacrosanct!

Reply:
>  Sometimes I take what you say as sacrosanct!

Please don't :P